contracts/ChipRegistry.sol

```diff
@@ -366,7 +352,7 @@ contract ChipRegistry is IChipRegistry, ClaimedPBT, Ownable {
      *
      * @param _ers                       Address of the ERS contract
      * @param _servicesRegistry          Address of the ServicesRegistry contract
-     * @param _developerRegistry               Address of the DeveloperRegistry contract
+     * @param _developerRegistry         Address of the DeveloperRegistry contract
      */
     function initialize(IERS _ers, IServicesRegistry _servicesRegistry, IDeveloperRegistry _developerRegistry) external onlyOwner {
```

Comment:
I have a PR to remove `_servicesRegistry` since there isn't really a system contract for it now doesn't make sense to store it